0008-atomic-registry.md

```diff
@@ -0,0 +1,144 @@
+                                          *Image created with new tag*
+                                      ->  thomasmckay/hammer:v1
+   ```
+
+   In this way, a user may pull an image (eg. thomasmckay/hammer:latest), manipulate it, then make it available with a new tag (eg. thomasmckay/hammer:v1). More than one Katello repo may push to a single project/imagestream, each supplying their own tags.  
+   :question: Is this too complicated and unnecessary? Atomic Registry doesn't support this.
+   :sparkles: Specify which tags are pushed to Atomic Registry and also which tags are synced into Katello.  
+   :sparkles: When creating a repo, tags should be configurable for sync or not, push or not, automatic or manual push.  
+   :sparkles: Defaults for above should be set per repo to handle new tags as they appear from a sync.  
+   :sparkles: Repositories in content views should also be configurable in this way.  
+   :question: Should tags be configurable? Could Sally's hammer:latest be synced as hammer:sally? Could a Katello tag hammer:13-test be pushed to Atomic Registry as hammer:latest-test?
+   :sparkles: Roles should be granular to the tag level to determine who may sync into or push out of a Katello repository.  
+   :sparkles: A central location should be available listing all Atomic Registry project/imagestream and their dependent repositories and settings. All should be searchable and configurations changed from there.  
+
+# User Authentication and Role Based Access Control (RBAC)
+[userauthentication]: #userauthentication
+
+Both Katello and Atomic Registry have the concept of a user, as well as the ability to assign roles and permissions to control access to actions and resources. At this time they are separate systems and as such there is no model for a "single sign on" (SSO) that would allow a user in the Katello UI to also be logged into the Atomic Registry UI.
+
+### User Authentication
+
+Katello uses either its own builtin authentication or LDAP ([Foreman LDAP Authentication](https://theforeman.org/manuals/1.12/index.html#4.1.1LDAPAuthentication)). Atomic Registry leverages openshift/origin authentication and is able to run in several modes, including LDAP ([Configuring Authentication and User Agent](http://docs.projectatomic.io/registry/latest/install_config/configuring_authentication.html)). Using LDAP would allow a closer feel to SSO but only because the user names would be consistent. Each tool would need those users configured individually.
+
+### Role Based Access Control (RBAC)
+
+
+Katello has a rich, fine-grained RBAC model ([Roles and Permissions](https://theforeman.org/manuals/1.12/index.html#4.1.2RolesandPermissions)) that ties resources to actions. For example, a permission to view and sync products could be assigned to a user to allow them to bring in updated content but that would prevent the creation or deletion of products. Further refinement is possible by adding a search filter.
```

Comment:
While it's true plain docker registry does not, Docker Enterprise does provide this. Atomic Registry brings some as well with admin, push&pull, and pull-only roles.

I agree this is an important part of what Katello w/ Atomic Registry can provide; I'll add more details and comparisons. 

Thanks!